app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminCheckProductOnProductGridActionGroup.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>	
+<!--	
+ /**	
+  * Copyright © Magento, Inc. All rights reserved.	
+  * See COPYING.txt for license details.	
+  */	
+-->	
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"	
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">	
+    <actionGroup name="AdminCheckProductOnProductGridActionGroup">	
+        <annotations>	
+            <description>Check the checkbox for the product on the Product Grid</description>	
+        </annotations>	
+        <arguments>	
+            <argument name="product" type="entity"/>	
+        </arguments>	
+
+      <checkOption selector="{{AdminProductGridSection.productRowCheckboxBySku(product.sku)}}" stepKey="selectProduct"/>    	
```

Comment:
Hi @AnnaAPak. Still, there are some issues with the indentation here and there. Please, double-check your changes. There's an option for auto code formatting in the modern editors and IDEs. I would recommend running the format command for your changes before the commit. 
It's easy to miss something like that mechanically but the mentioned automation will help you to avoid that. 

Thanks! 

```suggestion
        <checkOption selector="{{AdminProductGridSection.productRowCheckboxBySku(product.sku)}}" stepKey="selectProduct"/>    	
```

Reply:
Should be fixed now. Thank you for the advice.